src/pages/organize/[orgId]/projects/[campId]/emails/[emailId]/compose.tsx

```diff
@@ -58,13 +59,21 @@ const EmailPage: PageWithLayout<Props> = ({ emailId, orgId }) => {
     return null;
   }
 
+  const now = new Date();
+  const readOnly =
+    !!email.published || (!!email.published && new Date(email.published) > now);
```

Comment:
This logic seems incorrect to me. If I read it correctly, it will make the email readonly if:

1. There is a publish date
2. OR there is a publish date and it's in the future

Isn't it enough with just one of these? I believe the first one is enough – the email should be readonly if it's been scheduled, regardless of whether the date is in the future.

Reply:
Sounds very correct! Will change to this!